Packs/AzureSecurityCenter/Integrations/MicrosoftDefenderForCloudEventCollector/README.md

```diff
@@ -0,0 +1,50 @@
+XSIAM collector for Microsoft Defender for Cloud alerts.
+This integration was integrated and tested with version xx of Microsoft Defender for Cloud Event Collector
+
+## Configure Microsoft Defender for Cloud Event Collector on Cortex XSIAM
+
+1. Navigate to **Settings** > **Configurations** > **Data Collection** > **Automations & Feed Integrations**.
+2. Search for Microsoft Defender for Cloud Event Collector.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Microsoft Azure Management URL |  | False |
+    | ID |  | True |
+    | Token |  | True |
+    | Key |  | True |
+    | Certificate Thumbprint | Used for certificate authentication. As appears in the "Certificates & secrets" page of the app. | False |
+    | Private Key | Used for certificate authentication. The private key of the registered certificate. | False |
+    | Subscription ID to use |  | True |
+    | Trust any certificate (not secure) |  | False |
+    | Use system proxy settings |  | False |
```

Comment:
Add missing description

Reply:
This is auto-generated from the yml added description where it is not self-explanatory.